apps/chatbots/views.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.decorators import permission_required
+from django.views.generic import CreateView
+
+from apps.chatbots.tables import ChatbotTable
+from apps.experiments.models import Experiment
+from apps.experiments.views.experiment import BaseExperimentView
+from apps.teams.decorators import login_and_team_required
+from apps.utils.BaseTableView import BaseTableView
+from apps.utils.helpers import generic_home
+
+
+@login_and_team_required
+@permission_required("chatbots.view_chatbot", raise_exception=True)
```

Comment:
Django permissions are based on Django app names and their DB models. Since there is no `ChatBot` DB model there also isn't a `view_chatbot` permission. This should use the same permissions as the current experiment list view `experiments.view_experiment`